src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
one more thing, you can remove prefix `ball` from the names of the variables, since they are already members of the Ball class :)